### PR TITLE
DOCSP-36060-fix-typo

### PR DESCRIPTION
**Pull request by rustagir**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-node/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-36060>
Staging - https://preview-mongodbrustagir.gatsbyjs.io/node/DOCSP-36060-limit-typo-fix/fundamentals/crud/read-operations/limit/#limit

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [ ] Are all the links working?
- [ ] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/fundamentals/crud/read-operations/limit.txt [hunk 1]
```diff
@@ -51,8 +51,9 @@ Limit
 
 The following example queries the collection to return the top three
 longest books. It matches all the documents with the query, applies
```
Comment (rachel-mack) [inline source/fundamentals/crud/read-operations/limit.txt:53]: Suggestion: I think this is a little clearer
```suggestion
longest books. It finds all documents that match the query, applies
```

File: source/fundamentals/crud/read-operations/limit.txt [hunk 2]
```diff
@@ -51,8 +51,9 @@ Limit
...
-a ``sort`` on the ``length`` field to return books with longer lengths before
-books, and applies a ``limit`` to return only ``3`` results:
+a descending ``sort`` on the ``length`` field to longer books before
+shorter books, and then applies a ``limit`` to return only the ``3``
+first results:
```
Comment (rachel-mack) [inline source/fundamentals/crud/read-operations/limit.txt:56]:
```suggestion
a descending ``sort`` on the ``length`` field to return longer books before
shorter books, and then applies a ``limit`` to return only the ``3``
first results:
```

**Review by rachel-mack — CHANGES_REQUESTED** [review]
1 suggestion, 1 change:

File: source/fundamentals/crud/read-operations/limit.txt (hunk 2 above)
Comment (rustagir) [reply source/fundamentals/crud/read-operations/limit.txt:56]: oops

**Review by rachel-mack — APPROVED** [review]
LGTM

**Pull request merged**